new docs readme, quickstart, sample_notebook (#1231)
spacetelescope/jdaviz: docs/quickstart.rst

In a Jupyter Notebook
---------------------

The power of ``jdaviz`` is that it can integrated into your Jupyter notebook workflow::

    from jdaviz import Specviz

    specviz = Specviz()
    specviz.app

> 
> ``jdaviz`` also provides a directory of `sample notebooks :ref:`sample_notebook` to test the application, located in the ``notebooks`` sub-directory

Reviewer: ```suggestion
``jdaviz`` also provides a directory of :ref:`sample notebooks <sample_notebook>` to test the application, located in the ``notebooks`` sub-directory
```

> `specviz <https://jdaviz.readthedocs.io/en/latest/specviz/import_data.html>`_, `cubeviz <https://jdaviz.readthedocs.io/en/latest/cubeviz/import_data.html>`_, `mosviz <https://jdaviz.readthedocs.io/en/latest/mosviz/import_data.html>`_, or `imviz <https://jdaviz.readthedocs.io/en/latest/imviz/import_data.html>`_ tools.

Reviewer 2: Should use `:ref:` instead of hardcoding the URL. Not all pages mentioned is set up for `:ref:`, so that needs to be fixed too as part of this work.

> ``jdaviz`` also provides a directory of `sample notebooks <https://jdaviz.readthedocs.io/en/latest/sample_notebooks.html>`_ to test the application, located in the ``notebooks`` sub-directory

Reviewer 2: ```suggestion
``jdaviz`` also provides a directory of `sample notebooks :ref:`sample_notebook` to test the application, located in the ``notebooks`` sub-directory
```

In a Jupyter Notebook
---------------------

The power of ``jdaviz`` is that it can integrated into your Jupyter notebook workflow::

    from jdaviz import Specviz

    specviz = Specviz()
    specviz.app

> To learn more about the various ``jdaviz`` application configurations and loading data, see the
> `specviz <https://jdaviz.readthedocs.io/en/latest/specviz/import_data.html>`_, `cubeviz <https://jdaviz.readthedocs.io/en/latest/cubeviz/import_data.html>`_, `mosviz <https://jdaviz.readthedocs.io/en/latest/mosviz/import_data.html>`_, or `imviz <https://jdaviz.readthedocs.io/en/latest/imviz/import_data.html>`_ tools.

Reviewer: ```suggestion
```
